include/deal.II/numerics/vector_tools.h

```diff
@@ -1858,41 +1899,41 @@ namespace VectorTools
    *
    * See the general documentation of this namespace for further information.
    */
-  template <int dim, int spacedim>
+  template <int dim, int spacedim, typename VectorType>
   void create_right_hand_side (const Mapping<dim, spacedim>    &mapping,
                                const DoFHandler<dim,spacedim>  &dof,
                                const Quadrature<dim>           &q,
                                const Function<spacedim,double> &rhs,
```

Comment:
since you added new template, the `double -> typename VectorType::value_type`. Same everywhere else.